controllers/hazelcast/hazelcast_status.go

```diff
@@ -127,7 +127,7 @@ func update(ctx context.Context, c client.Client, h *hazelcastv1alpha1.Hazelcast
 
 	cl, ok := GetClient(types.NamespacedName{Name: h.Name, Namespace: h.Namespace})
 
-	if ok && cl.client != nil && cl.client.Running() {
+	if ok && cl.client != nil && util.IsClientConnected(hazelcast.NewClientInternal(cl.client)) {
```

Comment:
This check would be easier to read. 
```suggestion
	if ok && cl.IsClientConnected() {
```

Reply:
You are right, I will change it to that.